test/performance/specs/site-editor.spec.js

```diff
@@ -187,6 +189,58 @@ test.describe( 'Site Editor Performance', () => {
 			}
 		} );
 	} );
+
+	test.describe( 'Navigating', () => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentythree' );
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyone' );
+		} );
+
+		const iterations = 5;
```

Comment:
Will 5 be enough to produce a stable output?

Reply:
Well it did for me, to be honest, I feel all the iterations we have in other metrics might not need to be high either. 

It was stable on the first run in this PR, I restarted to check again.

Reply:
Why not test using twenty twenty four?

Reply:
@draganescu we don't have it in our testing environment (a fixed version) like we do for this one which we're already using in other tests.